src/Tools/Md2Html/Md2HtmlSanitizer.cs

```diff
@@ -20,8 +21,24 @@ internal Md2HtmlSanitizer()
 
             AllowedAttributes.Add(@"content");
             AllowedAttributes.Add(@"http-equiv");
+            AllowedAttributes.Add(@"id");
 
             AllowedCssProperties.Add(@"src");
+            AllowedCssProperties.Add(@"word-break");
+            AllowedCssProperties.Add(@"word-wrap");
+            AllowedCssProperties.Add(@"-moz-tab-size");
+            AllowedCssProperties.Add(@"-o-tab-size");
+            AllowedCssProperties.Add(@"tab-size");
+            AllowedCssProperties.Add(@"-webkit-hyphens");
+            AllowedCssProperties.Add(@"-moz-hyphens");
+            AllowedCssProperties.Add(@"-ms-hyphens");
+            AllowedCssProperties.Add(@"hyphens");
+            AllowedCssProperties.Add(@"background-position-x");
+            AllowedCssProperties.Add(@"background-position-y");
+
+            AllowedSchemes.Add(@"file");
```

Comment:
this is for images loading?

Reply:
Yes

Reply:
these look fine to me in general - I guess I would like us to do a bit of research on "file" to see if this could somehow be used to load and execute arbitrary js - to me it looks like you'd need another script tag to do that, but I'm not sure.

Reply:
We only need this for loading images from markdown files. I think markdown do support embedded images so we could make that an requirement for all images? We do not allow script tags at all so there is no way to misuse `file` there. I will do some digging to see if there is a way to load js outside a script tag.